xapian-core/include/xapian/mset.h

```diff
@@ -47,6 +47,10 @@ class XAPIAN_VISIBILITY_DEFAULT MSet {
     // Helper function for fetch() methods.
     void fetch_(Xapian::doccount first, Xapian::doccount last) const;
 
+  private:
```

Comment:
Isn't this already `private`? There's nothing to change from the default for class members.